sqs/src/main/scala/akka/stream/alpakka/sqs/SqsSourceStage.scala

```diff
@@ -75,7 +75,7 @@ final class SqsSourceStage(queueUrl: String, settings: SqsSourceSettings)(implic
         currentRequests = currentRequests - 1
         maxCurrentConcurrency = if (result.getMessages.isEmpty) 1 else maxConcurrency
 
-        val receivedMessages = result.getMessages.asScala.reverse
```

Comment:
I still wonder how it got there...

Reply:
I think it was there from the very beginning of the SQS connector. Anyways, this library fixes more issues than it causes 😁😉